about.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About WL Global Summit</title>
+    <link
+    rel="stylesheet"
+    href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css"
+  />
+  <link rel="stylesheet" href="about.css">
+  </head>
+  <body>
+    <!-- header start -->
+    <div
+    class="uk-background-norepeat uk-background-cover uk-background-center-right uk-section"
+  >
+    <div
+      class="uk-position-cover"
+      
+    >
+      <div class="nav-wrapper">
+        <nav class="nav flex-row w-100">
+          <svg
+            id="btn"
+            class="nav_botton hamb hamRotate ham"
+            viewBox="0 0 100 100"
+            width="60"
+          >
+            <path
+              class="line top"
+              d="m 30,33 h 40 c 3.722839,0 7.5,3.126468 7.5,8.578427 0,5.451959 -2.727029,8.421573 -7.5,8.421573 h -20"
+            />
+            <path class="line middle" d="m 30,50 h 40" />
+            <path
+              class="line bottom"
+              d="m 70,67 h -40 c 0,0 -7.5,-0.802118 -7.5,-8.365747 0,-7.563629 7.5,-8.634253 7.5,-8.634253 h 20"
+            />
+          </svg>
+          <div class="logo-wrapper">
+            <a href="/"
+              ><img
+                width="150px"
+                height="auto"
+                src="./image/logo.png"
+                alt="logo"
+            /></a>
+          </div>
+          <div class="menu-container" id="menu-show">
```

Comment:
- [ ] Good Job for fixing the menu links as the reviewer told you but it would be better to fix the about link here as it is sending the user to the home page instead of the about page you can add another link to the menu which is the home page link as it will be better for the user experience.
![image](https://user-images.githubusercontent.com/74744787/212487427-cb2f97ae-15bf-4fee-8d96-1509df00c60f.png)